src/Integration.Vsix.UnitTests/InfoBar/InfoBarManagerTests.cs

```diff
@@ -204,6 +204,8 @@ public void InfoBarManager_DetachInfoBar()
 
         #region Test helpers
 
+        private static SonarLintImageMoniker CreateFromVsMoniker(dynamic imageMoniker) => new SonarLintImageMoniker(imageMoniker.Guid, imageMoniker.Id);
```

Comment:
`KnownMonikers` is a very problematic class to pass as argument, as it creates ambiguity between `ImageMoniker` from `Microsoft.VisualStudio.ImageCatalog` assembly and `ImageMoniker` from `Microsoft.VisualStudio.Imaging.Interop`. Since it's test code, I opted to just use dynamic.

Reply:
You don't need 'dynamic'; just replace it here with the fully qualified type name i.e. `Microsoft.VisualStudio.Imaging.Interop.ImageMoniker`.

Reply:
It's the exact same namespace, but different assemblies.

Reply:
The test built and ran fine for me locally using the fully qualified type name instead of dynamic. It also worked if I added 'using Microsoft.VisualStudio.Imaging.Interop;'.
Does it fail for you locally, or in a CI build?

Reply:
Odd; it fails for me locally. And it makes sense, since both assemblies are indeed referenced.
Weird that it works for you 🤔 
![image](https://user-images.githubusercontent.com/60586879/97449176-b1157f00-1931-11eb-8597-ff098e81507e.png)

Reply:
Edit: I see now that despite this error, the code actually compiles and the tests pass, that's very odd 🤔 
So I will check-in it with the namespace and just ignore the editor errors.

Reply:
No. Could be an incorrect IntelliSense cache on your machine; that would cause those symptoms.